hash/msx1_cart.xml

```diff
@@ -18345,6 +18384,399 @@ legacy FM implementations cannot find it.
 
 <!-- Homebrew / Doujin software -->
 
+	<software name="labbaye">
+		<description>l'Abbaye des Morts (v1.2)</description>
+		<year>2020</year>
+		<publisher>&lt;homebrew&gt;</publisher>
+		<info name="developer" value="RetroDeluxe"/>
+		<part name="cart" interface="msx_cart">
+			<feature name="slot" value="ascii8" />
+			<dataarea name="rom" size="0x40000">
+				<rom name="l'abbaye des morts - retrodeluxe (2020) (v1.2).rom" size="0x40000" crc="c5aef269" sha1="3b417dd6e2debf80f898e681f1c6a754dfaed2e1" />
+			</dataarea>
+		</part>
+	</software>
```

Comment:
Is this really “homebrew”?  It had a commercial physical release, even though it’s long after the commercial life of the system: https://cotegamers.com/shop/en/accueil/144-abbaye-des-morts-msx.html

Reply:
Good question.

The released roms are on github: https://github.com/retrodeluxe/AbbayeMSX/releases

I do not know if the physical release was made from one of these released roms or from a special enhanced build (like with most msx software released through Matra).

Reply:
Was it always on GitHub, or did they release binaries there some time after the physical game went on sale?  It’s been three years now anyway, so it’s clear to add to MAME either way.

Reply:
I don't know if it's the case with this one, but often, a later MSX cartridge release has improvements over the free version.

Reply:
According to archive.org cotegamers started taking pre-orders in December 2022: https://web.archive.org/web/20221229120854/https://cotegamers.com/shop/en/accueil/144-abbaye-des-morts-msx.html

So the github releases were before that physical release.

Reply:
So we have:
- the v1.1 rom which was submitted to MSXDev
- the v1.2 rom which was released on github in 2020
- a physical release in early 2023 which may or may not contain the v1.2

According to our 3 year rule the physical version should not be added until early 2026. Assuming that the physical release contains the v1.2 version I should remove that from this PR (even though the rom has already been available for 3 years on github) ?

Reply:
If v1.2 was released online in 2020, then it can be added whether the physical release this year contains the same version or not.  It’s still been in distribution for three years.

Other ports of l'Abbaye des Morts (e.g. the one for Mega Drive) were released simultaneously as donationware to download as well as physical cartridges for sale with identical content.